config/config.go

```diff
@@ -904,10 +908,11 @@ type MempoolConfig struct {
 // DefaultMempoolConfig returns a default configuration for the CometBFT mempool.
 func DefaultMempoolConfig() *MempoolConfig {
 	return &MempoolConfig{
-		Type:      MempoolTypeFlood,
-		Recheck:   true,
-		Broadcast: true,
-		WalPath:   "",
+		Type:           MempoolTypeFlood,
+		Recheck:        true,
+		RecheckTimeout: 300 * time.Millisecond,
```

Comment:
This might be too little for heavy app implementations for `CheckTx`

Reply:
That's right. I changed it to 1 second. I also added a paragraph to `config.toml.md` explaining how to roughly calculate this value.